src/utils/get-edge-path/get-edge-path.ts

```diff
@@ -11,8 +12,10 @@ export function getEdgePath(): string {
       return '%windir%/SystemApps/Microsoft.MicrosoftEdge_8wekyb3d8bbwe/MicrosoftEdge.exe';
 
     case 'Linux':
-      // Need help to get default install path on Linux. PR is welcomed
-      throw new Error('You should supply the path to the Edge App in the launch options');
+      if (which.sync('microsoft-edge', { nothrow: true })) {
+        return which.sync('microsoft-edge');
+      }
+      throw new Error('Can not find installed "microsoft-edge"');

```

Comment:
- Unless I am wrong, it seems to have a typo in `Can not` , shouldn't it be `Cannot` ?
 - Your error message is great, but in the context of a library or an API, an error message should always gives a hint to the developer on how to solve the problem : it makes the lib self documented and the consumer can learn how to use it by failing fast.
 my proposal: `Cannot find install path for Microsoft Edge. Either you can install it by running the command 'npx playwright install msedge', or you should provide the path to the Edge App in the launch options.`